storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,39 @@
+[id="persistent-storage-csi-azure-disk"]
+= Azure Disk CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
+
+:FeatureName: Azure Disk Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Azure Disk storage assets, {product-title}, after this feature is enabled, installs the Azure Disk CSI Driver Operator and the Azure Disk CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
Technically, the sentence seems correct but the commas/clauses trip me up a bit. Perhaps a bit of a rearrangement might help? Something like:
> To create CSI-provisioned PVs that mount to Azure Disk storage assets with this feature enabled, {product-title} installs the Azure Disk CSI Driver Operator and the Azure Disk CSI driver by default in the `openshift-cluster-csi-drivers` namespace.

Or could you just leave out the "this feature is enabled" part altogether?

Reply:
Nope! You cannot leave out the "feature enabled" part. Engr/QE were very definite that I had to stress that the feature has to be enabled or you don't get anything. I used your first option. Thanks!